src/main/scala/com/lightbend/kafkalagexporter/InfluxDBPusherSinkConfig.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package com.lightbend.kafkalagexporter
+
+import com.typesafe.config.Config
+
+class InfluxDBPusherSinkConfig(sinkType: String, metricWhitelist: List[String], config: Config) extends SinkConfig(sinkType, metricWhitelist, config)
+{
+    val endpoint: String = config.getString("reporters.influxDB.endpoint")
+    val port: Int = config.getInt("reporters.influxDB.port")
+    val defaultDatabase: String = "kafka_lag_exporter"
+    val database: String = if (config.hasPath("reporters.influxDB.database")) config.getString("reporters.influxDB.database") else defaultDatabase
+    val username: String = if (config.hasPath("reporters.influxDB.username")) config.getString("reporters.influxDB.username") else ""
+    val password: String = if (config.hasPath("reporters.influxDB.password")) config.getString("reporters.influxDB.password") else ""
+    val async: Boolean = if (config.hasPath("reporters.influxDB.async")) config.getBoolean("reporters.influxDB.async") else true
```

Comment:
Is there a reason to support synchronous writing?

Reply:
It would be better and more generic to support both the modes as supported by InfluxDB. Users can always pick and chose as per their needs. What do you think ?

Reply:
I'm not certain why we should expose this configuration to the end user if Kafka Lag Exporter is doing the writing.

Reply:
Yes, Kafka Lag Exporter is handling the InfluxDB writes. But, InfluxDB supports writes in both synchronous and asynchronous mode. So, we thought that it will be generic and flexible to provide the end users the option to pick their desired mode based on their deployments, with default as `async` mode (usually the preferred choice). Also, this is an optional configuration. 

Please let us know if this clarifies the approach.

Reply:
Got it. Sounds good to me.